src/emptyState/EmptyState.tsx

```diff
@@ -7,14 +7,23 @@ import { StyledEmptyState } from './EmptyState.style'
 export type EmptyStateProps = Readonly<{
   className?: string
   image: string
-  text: string
+  text: string | JSX.Element
```

Comment:
You can use `text: React.ReactNode` which includes both `string` and React elements but is more flexible

Reply:
Unfortunately the story page sends me a "too much recursion" error if I use `React.ReactNode`